modules/renter.go

```diff
@@ -52,6 +52,7 @@ type FileInfo struct {
 	Filesize       uint64            `json:"filesize"`
 	Available      bool              `json:"available"`
 	Renewing       bool              `json:"renewing"`
+	Redundancy     float64           `json:"redundancy"`
 	UploadProgress float64           `json:"uploadprogress"`
```

Comment:
is it redundant to have both `Redundancy` and `UploadProgress`? I feel like they are the same thing. If you are always displaying the progress of the minimal chunk, then you have the progress of everything has a whole.

Or, do they have different behaviors? If they do, this should be documented somewhere.